rvv-intrinsic-rfc.md

```diff
@@ -246,6 +249,30 @@ size_t vsetvlmax_e8m8 ();
 
 There is no need to specify the behavior of tail and masked-off elements being undisturbed or agnostic. The default setting is tail agnostic and masked-off undisturbed. If users do not want to keep the values in masked-off elements, they could pass `vundefined()` as the `maskedoff` value.
 
+`SEW` and `LMUL` are a part of the naming. They are static information for the
+intrinsics.
+
+All of the intrinsic function having a `vl` argument to specify the active
+vector length, except few functions are work with fixed length, e.g. `vmv.x.s`,
+`vfmv.f.s` and whole register move/load/store.
```

Comment:
1. IIRC,  whole register intrinsic function are not supported, shuld we mention them?
2. Could you list all vl-less intrinsic functions? I had list all in https://github.com/riscv/rvv-intrinsic-doc/issues/42 (If I did not miss something..)

Reply:
Thanks! updated the list.